packages/base/Grid/src/GridItem/GridItem.tsx

```diff
@@ -29,33 +28,61 @@ export interface Props
   children?: ReactNode
 }
 
-const useStyles = makeStyles<Theme>(styles, {
-  name: 'PicassoGridItem',
-})
+const gridSpacingMapping: { [K in GridSpacing]: string } = {
+  '0': 'p-[0px]',
+  '8': 'p-[4px]',
+  '16': 'p-[8px]',
+  '24': 'p-[12px]',
+  '32': 'p-[16px]',
+  '64': 'p-[32px]',
+  '72': 'p-[36px]',
+  '80': 'p-[40px]',
```

Comment:
Stylistic NIT: quotes are unnecessary here
```suggestion
  0: 'p-[0px]',
  8: 'p-[4px]',
  16: 'p-[8px]',
  24: 'p-[12px]',
  32: 'p-[16px]',
  64: 'p-[32px]',
  72: 'p-[36px]',
  80: 'p-[40px]',
```